internal/tiltd/tiltd.go

```diff
@@ -4,12 +4,13 @@ import (
 	"context"
 	"fmt"
 	"strings"
+	"io"
 )
 
 const Port = 10000
 
 type TiltD interface {
-	CreateService(ctx context.Context, k8sYaml string, dockerFileText string, mounts []Mount, steps []Cmd, dockerfileTag string) error
+	CreateService(ctx context.Context, k8sYaml string, dockerFileText string, mounts []Mount, steps []Cmd, dockerfileTag string, stdoutWriter io.Writer, stderrWriter io.Writer) error
```

Comment:
imo you could get away with calling these e.g. `stdout` rather than `stdoutWriter` (though i have no real justification behind this except that if I were reading this code, i'd assume `stdout` was a writer unless otherwise noted)

also, if you have multiple args of the same type, can notate them as: `... stdout, stderr io.Writer`

Reply:
yeah, that's much prettier, thanks